django-backend/src/api/data/fdg_output_file.bak

```diff
@@ -0,0 +1,216 @@
+'icij', (141824, 1264)

```

Comment:
should this be committed? I assume it's a generated file.

Reply:
The script (main.py) in the data-formatter directory builds the shelve database from the JSON data. These files (*.bak and *.dir) are created during this process. Shelve uses any one of the 4 different DB implementations depending on the machine and other factors. I think these files should be indicating something about it. I wasn't able to find the actual purpose of these files so far and thought it would be wise just to include it.

Reply:
I have put it in gitignore and removed them.